pages/1-stripe-integration.html

```diff
@@ -0,0 +1,364 @@
+<!doctype html>
```

Comment:
@rootical can it be moved under `/showcases/` path? The full url should be http://localhost:1234/showcases/1-stripe-integration